miner/Miner.hs

```diff
@@ -208,107 +191,114 @@ main = do
     opts = info (pClientArgs <**> helper)
         (fullDesc <> progDesc "The Official Chainweb Mining Client")
 
-scheme :: Env -> (TargetBytes -> HeaderBytes -> IO HeaderBytes)
-scheme env = case cmd $ args env of
-    CPU e -> cpu (version $ args env) e (gen env)
-    GPU _ -> gpu
+run :: RIO Env ()
+run = do
+    logInfo "Starting Miner."
+    env <- ask
+    case cmd $ args env of
+        GPU -> logError "GPU mining is not yet available."
```

Comment:
If it's not working, we should take this out of the command-line help.  We shouldn't ship non-working features.

Reply:
I'll comment out (not delete) all the things that Greg will soon need, so that he can jump right in.